package.json

```diff
@@ -37,6 +37,14 @@
                     "light": "resources/light/filter.svg"
                 }
             },
+            {
+                "command": "aks.refreshSubscription",
+                "title": "Refresh subscription...",
```

Comment:
Title case, and no ellipsis unless it needs additional data to complete (which seems unlikely).